.github/workflows/charts-release.yaml

```diff
@@ -88,7 +88,7 @@ jobs:
           CHARTS=(${{ steps.collect-changes.outputs.modifiedChartsAfterTag }})
           echo "Removing Chart Docs prior to regeneration..."
           for deltrain in stable SCALE incubator games enterprise develop non-free deprecated dependency core; do
```

Comment:
We can also grab the train folders here instead of hardcoding (which we should actually do in more places tbh)